src/data/pokemon_graphics/unused_anims.h

```diff
@@ -0,0 +1,1829 @@
+static const union AnimCmd *const sUnusedAnims_Groudon[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_Rayquaza[] = {sAnim_GeneralFrame0, sUnusedAnim_Rayquaza_1, sUnusedAnim_Rayquaza_2};
+static const union AnimCmd *const sUnusedAnims_Latias[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_Latios[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_Jirachi[] = {sAnim_GeneralFrame0, sUnusedAnim_Jirachi_1};
+static const union AnimCmd *const sUnusedAnims_Deoxys[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_Chimecho[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_Egg[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownC[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownD[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownE[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownF[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownG[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownH[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownI[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownJ[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownK[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownL[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownM[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownN[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownO[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownP[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownQ[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownR[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownS[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownT[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownU[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownV[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownW[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownX[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownY[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownZ[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownEMark[] = {sAnim_GeneralFrame0};
+static const union AnimCmd *const sUnusedAnims_UnownQMark[] = {sAnim_GeneralFrame0};
+
+#define ANIM_CMD(name, anims)      [SPECIES_##name] = sUnusedAnims_##anims
```

Comment:
Same as the other file, we can write this out instead of using a macro (if anything, the macro just makes it harder to search for where the species constant or anim table is used)

Reply:
Right, I forgot to apply those changes here too.